src/log/log_functions.c

```diff
@@ -335,6 +335,8 @@ void Cleanup(void)
 		c->clean();
 		c = c->next;
 	}
+
+	flush_all_logs(true /*close_fd*/);
```

Comment:
Cleanup is called before exit and before nfs ganesha admin thread stops for some reason. So, we need to flush logs before exiting.